db/controllers/controllers.js

```diff
@@ -67,10 +68,23 @@ const postComment = (req, res, next) => {
       next(err);
     });
 };
+
+const getVotes = (req, res, next) => {
```

Comment:
Are we GETting here? What should this function do, if successful? You should name it to reflect that